app-timer-edit/src/main/java/xyz/aprildown/timer/app/timer/edit/UpdateStepDialog.kt

```diff
@@ -179,6 +182,35 @@ class UpdateStepDialog :
                     addScreenItems(
                         context = context,
                         action = current.toScreenAction(),
+                        onPickImageClick = {
+                            registerForActivityResult(
```

Comment:
Registering a launcher after onCreate will crash the app. If I remember correctly, you must create the launcher before or in onCreate.

Reply:
I indeed noticed that with "EditActivity.kt", which has this fixed. I leaved it that way here because I didn't know how to accommodate the difference between this file and the other. Beside, I still don't know when this file intervenes in the app